### DISCUSSION TITLE
RecordCreator.Create issue

### DISCUSSION THREAD

**Discussion by sherwin-azility** [JoshClose/CsvHelper#2266] [Q&A]
We are seeing the below exception thrown when running ` var records = csv.GetRecords<T>();` in our test environments. The csv is working fine in my local machine. Tried to do the one record per read way and is throwing the same issue. Anything to check in our test server? We are using the latest table version 32.0.3

```
Object reference not set to an instance of an object. at lambda_method(Closure ) at CsvHelper.Expressions.RecordCreator.Create[T]()
```

**Comment by JoshClose** [comment]
What is the difference between your local machine and the test server?

**Reply by sherwin-azility** [reply]
Hi @JoshClose, I think the issue within the end column of my csv that have empty values. Which is not even mapped on T
I tried to modify the Last column with "" values and it works fine, I wonder why having empty values is working on my local though. 

Column1, LastColumn
1,
1,
2,
